bin/helpers.js

```diff
@@ -8,23 +8,84 @@ export async function writeRollup(inputOptions, outputOptions) {
   await bundle.close();
 }
 
+export async function withEsmFile(esmSourcePath, fn) {
+  const esmTempPath = path.join(
+    path.dirname(esmSourcePath),
+    "tmp-esm-index.ts"
+  );
+
+  const lines = (await fs.readFile(esmSourcePath, "utf8")).split(/\r?\n/);
+  const startLine = lines.findIndex((line) =>
+    line.includes("!helmet-start-of-commonjs-exports")
+  );
+  const endLine = lines.findIndex((line) =>
+    line.includes("!helmet-end-of-commonjs-exports")
+  );
+  const lineCount = endLine - startLine + 1;
+  lines.splice(startLine, lineCount);
+
+  try {
+    await fs.writeFile(esmTempPath, lines.join("\n"));
+
+    await fn(esmTempPath);
+  } finally {
+    await fs.unlink(esmTempPath);
+  }
+}
+
 export async function withCommonJsFile(esmSourcePath, fn) {
-  const commonJsSourcePath = path.join(
+  const commonJsTempPath = path.join(
     path.dirname(esmSourcePath),
     "tmp-commonjs-index.ts"
   );
 
   const lines = (await fs.readFile(esmSourcePath, "utf8")).split(/\r?\n/);
-  const resultLines = lines.slice(
-    0,
-    lines.findIndex((line) => line.includes("!helmet-end-of-commonjs"))
+  const startLine = lines.findIndex((line) =>
+    line.includes("!helmet-start-of-commonjs-exports")
   );
+  const endLine = lines.findIndex((line) =>
+    line.includes("!helmet-end-of-commonjs-exports")
+  );
+  lines.splice(startLine, 1);
+  const resultLines = lines.slice(0, endLine);
 
   try {
-    await fs.writeFile(commonJsSourcePath, resultLines.join("\n"));
+    await fs.writeFile(commonJsTempPath, resultLines.join("\n"));
 
-    await fn(commonJsSourcePath);
+    await fn(commonJsTempPath);
   } finally {
-    await fs.unlink(commonJsSourcePath);
+    await fs.unlink(commonJsTempPath);
+  }
+}
+
+export async function renameFile(oldPath, newPath) {
+  await fs.rename(oldPath, newPath)
+    .catch((error) => {
+      if (error) console.error(error);
+    });
+}
```

Comment:
It seems strange that we'd drop errors here—I think I'd like it to crash. Could we delete this function and just use `fs.rename`?